terraform/modules/monitoring/dashboard.json

```diff
@@ -5,9 +5,8 @@
     "columns": 4,
     "tiles": [
       {
-        "height": 2,
         "widget": {
-          "title": "Cloud Run: Request Count for Justification Verification Server",
+          "title": "Cloud Run Revision: Request Count for Justification Verification Server",
```

Comment:
For Cloud Run Service and Cloud Run Revision, see docs here https://cloud.google.com/run/docs/managing/services and https://cloud.google.com/run/docs/managing/revisions.
I don't think we need to change the title here.

Reply:
The reason I changed it to Cloud Run Revision is that if you manually create a dashboard graph, the metric is selected from `Cloud Run Revision`, and the system suggest you to name the chart `Cloud Run Revision: xxxxx`.